src/components/FormComponent/FormComponent.css

```diff
@@ -0,0 +1,64 @@
+.form-style {
+  font: 14px Arial, Helvetica, sans-serif;
+  max-width: 400px;
+  margin: 10px auto;
+  padding: 16px;
+  background: #F7F7F7;
+}
+
+.form-input {
+  -webkit-transition: all 0.30s ease-in-out;
+  -moz-transition: all 0.30s ease-in-out;
+  -ms-transition: all 0.30s ease-in-out;
+  -o-transition: all 0.30s ease-in-out;
+  outline: none;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  width: 100%;
+  background: #fff;
+  margin-bottom: 15px;
+  border: 1px solid #ccc;
+  padding: 15px;
+  color: #555;
+  font: 14px Arial, Helvetica, sans-serif;
+}
+
+.form-checkbox {
+  -webkit-transition: all 0.30s ease-in-out;
+  -moz-transition: all 0.30s ease-in-out;
+  -ms-transition: all 0.30s ease-in-out;
+  -o-transition: all 0.30s ease-in-out;
+  outline: none;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  color: #555;
+  font: 14px Arial, Helvetica, sans-serif;
+  margin-bottom: 15px;
+  width: 200px;
+}
+
+.form-input:focus {
```

Comment:
This should be in separate css file, containing only css related to InputFieldComponent